source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisfirehose-s3/lib/index.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
```

Comment:
This is a build product, it shouldn't have gotten checked in. Check your .gitignore